analysis/deathknight/src/runeforges/RuneOfHysteria.tsx

```diff
@@ -91,39 +91,53 @@ class RuneOfHysteria extends Analyzer {
   suggestions(when: When) {
     when(this.efficiencySuggestionThresholds).addSuggestion((suggest, actual, recommended) =>
       suggest(
-        <span>
+        <Trans id="deathknight.shared.runeOfHysteria.suggestion.suggestion">
           Avoid being Runic Power capped at all times, you wasted{' '}
           {this.runicPowerWastedDuringHysteria} PR of {SPELLS.RUNE_OF_HYSTERIA.name} by being RP
           capped.
-        </span>,
+        </Trans>,
       )
         .icon(SPELLS.RUNE_OF_HYSTERIA.icon)
         .actual(
-          t({
-            id: 'deathknight.suggestions.hysteria.efficiency',
-            message: `You wasted ${formatPercentage(actual)}% of RP from ${
-              SPELLS.RUNE_OF_HYSTERIA.name
-            } by being RP capped.`,
-          }),
+          <Trans id="deathknight.shared.runeOfHysteria.suggestion.actual">
+            You wasted {formatPercentage(actual)}% of RP from {SPELLS.RUNE_OF_HYSTERIA.name} by
+            being RP capped
+          </Trans>,
         )
-        .recommended(`${formatPercentage(recommended)}% is recommended`),
+        .recommended(
+          <Trans id="deathknight.shared.runeOfHysteria.suggestion.recommended">
+            {formatPercentage(recommended)}% is recommended
+          </Trans>,
+        ),
     );
   }
 
   statistic() {
     let gainedSpell: React.ReactNode = <></>;
     let wastedSpell: React.ReactNode = <></>;
+    let runicPowerWastedDuringHysteriaText: React.ReactNode = <></>;
 
     if (this.selectedCombatant.spec === SPECS.BLOOD_DEATH_KNIGHT) {
       gainedSpell = (
-        <>
+        <Trans id="deathknight.shared.runeOfHysteria.statistic.gainedSpell">
           , resulting in {this.gainedDeathStrikes} additional {SPELLS.DEATH_STRIKE.name}
-        </>
+        </Trans>
       );
       wastedSpell = (
-        <>
+        <Trans id="deathknight.shared.runeOfHysteria.statistic.wastedSpell">
           , losing out on {this.wastedDeathStrikes} {SPELLS.DEATH_STRIKE.name}
-        </>
+        </Trans>
+      );
+    }
+
+    // Lingui doesn't fare well with nested Trans, maybe even in combination with conditionals
```

Comment:
I'm unsure myself if this is the right way to go about it, The maintainers might give a better 2nd opinion on this.

Reply:
To be fair, I've never worked with Lingui before, so maybe I'm missing some construct which it can work with.

Reply:
rather than using `<Trans />` for this, you may be able to use the `t(...)` macro directly. however, that may have its own issues with nesting. I do see it used in conditionals in (for example) `ReportDurationWarning.tsx`

Reply:
Yes, I've tried looking through the repo for `<Trans />` usages, and while I haven't looked through everything, just about.. 2/3s of it, nowhere I've seen nested `<Trans>` components. However using `t()` inside `<Trans>` seemed alright, so I've refactored and pushed changes with that.